tests/conftest.py

```diff
@@ -61,6 +62,7 @@ def init_env(tmp_path_factory, worker_id) -> None:
     # don't have several processes trying to initialize pgserver in parallel.
     root_tmp_dir = tmp_path_factory.getbasetemp().parent
     with FileLock(str(root_tmp_dir / 'pxt-init.lock')):
+        Env._init_env(reinit_db=True)
```

Comment:
Why do we need to call this explicitly now()?

Reply:
In order to set `reinit_db=True` when it gets called. Otherwise there can be stale db artifacts lying around when the tests start running, if they were previously interrupted with Ctrl-C.

I think this is related to the issue Jim's been running into.

Reply:
Leave that as a comment? It's not immediately obvious.

Reply:
Added.